liquibase-core/src/test/groovy/liquibase/changelog/DatabaseChangeLogTest.groovy

```diff
@@ -419,7 +443,7 @@ create view sql_view as select * from sql_table;'''
                 "com/example/not/fileX.sql"     : "file X",
         ])
         def changeLogFile = new DatabaseChangeLog("com/example/root.xml")
-        changeLogFile.includeAll("com/example/missing", false, null, true, changeLogFile.getStandardChangeLogComparator(), resourceAccessor, new ContextExpression(), new Labels(), false)
+        changeLogFile.includeAll("com/example/missing", false, null, true, changeLogFile.getStandardChangeLogComparator(), resourceAccessor, new ContextExpression(), new LabelExpression(), false)
```

Comment:
Shouldn't `LabelExpression` be `Labels`? As I understand it, the changelog cannot have a label expression (or filter). In looking at  [the includeAll case statement near this line in DatabaseChangelog.java](https://github.com/liquibase/liquibase/blob/63705dc90d775723d48859e33bdcbbb67c237572/liquibase-core/src/main/java/liquibase/changelog/DatabaseChangeLog.java#L519), it looks like Labels is the correct object type.  However ... this is working so maybe I just need some general education on when to use `Labels` versus `LabelExpression`.

Reply:
These tests are using the deprecated version of `includeAll` :

```
public void includeAll(String pathName,
                           boolean isRelativeToChangelogFile,
                           IncludeAllFilter resourceFilter,
                           boolean errorIfMissingOrEmpty,
                           Comparator<String> resourceComparator,
                           ResourceAccessor resourceAccessor,
                           ContextExpression includeContextFilter,
                           LabelExpression labelExpression,
                           boolean ignore)
```
Which we still use for backward compatibility. These tests were failing because passing labels (instead of label expression) would also require including these new parameters (minDepth/maxDepth), so at that moment I thought might be simpler to keep it that way, but I can update these tests and make use of the a non-deprecated includeAll method.